updated issue templates and added pull_request templates (#9345)
python/mypy: .github/issue_template/feature.md

---
name: 🚀 Feature

> about: Submit a proposal for a new feature

Reviewer: ```suggestion
about: Submit a proposal for a new mypy feature
```
Also, I wonder if we should add somewhere that this is *not* the place to propose new type system features -- those should generally be discussed on `typing-sig@python.org` or in https://github.com/python/typing/issues and probably have to go through the PEP process (see https://www.python.org/dev/peps/pep-0001/).

labels: "feature"
---

## 🚀 Feature

> (A clear and concise description of what the feature is.)

Reviewer: ```suggestion
(A clear and concise description of your feature proposal.)
```

## Pitch

> (Please explain why this feature should be implemented and how it would be used.)

Reviewer: Maybe add an encouragement to add examples.

---
name: 🚀 Feature
about: Submit a proposal for a new feature
labels: "feature"
---

## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines](https://github.com/python/mypy/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

> ## Motivation

Reviewer 2: I'm not sure I understand the difference between "Motivation" and "Pitch".

Author: @JelleZijlstra I have removed the Motivation section.
